policy/report.go

```diff
@@ -79,6 +79,8 @@ func (sd *ScoreDistribution) Add(score *Score) {
 		sd.D++
 	case ScoreRating_failed:
 		sd.F++
+	case ScoreRating_error:
```

Comment:
to cover all cases we also would need to catch the case where the score is unrated